README.md

```diff
@@ -69,7 +69,21 @@ The Krux code is a Python package that should be installed with [Poetry](https:/
 pip install poetry
 poetry install
 ```
-
+There may be a problem at this stage when trying to directly install Poetry, we have a solution proposal(this comand is for unix system mac or linux):
```

Comment:
I would only simplify this line to:
If you have a problem installing Poetry on Linux OS try:

Reply:
Ok good idea

Reply:
better?

Reply:
awesome! Now just wait for @odudex to merge the PR 😉